360-Videos-Metadata/360VideosMetadata.py

```diff
@@ -58,18 +58,27 @@
 # XML contents.
 rdf_prefix = " xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\" "
 
-spherical_xml = (
+spherical_xml_header = (
     "<?xml version=\"1.0\"?>\n"
     "<rdf:SphericalVideo"
     " xmlns:rdf=\"http://www.w3.org/1999/02/22-rdf-syntax-ns#\""
-    " xmlns:GSpherical=\"http://ns.google.com/videos/1.0/spherical/\">\n"
+    " xmlns:GSpherical=\"http://ns.google.com/videos/1.0/spherical/\">\n")
+
+spherical_xml_contents = (
     "  <GSpherical:Spherical>true</GSpherical:Spherical>\n"
     "  <GSpherical:Stitched>true</GSpherical:Stitched>\n"
     "  <GSpherical:StitchingSoftware>Spherical Metadata Tool"
     "</GSpherical:StitchingSoftware>\n"
     "  <GSpherical:ProjectionType>equirectangular"
-    "</GSpherical:ProjectionType>\n"
-    "</rdf:SphericalVideo>")
+    "</GSpherical:ProjectionType>\n")
+
+spherical_xml_contents_ou = ("  <GSpherical:StereoMode>over-under" +
```

Comment:
I'd recommend either expanding the ou/lr to over_under/left_right OR use python string formatting to keep the over_under/left_right-ness close to command line parsing

e.g., 

spherical_xml_contents = "  <GSpherical:StereoMode>%s</GSpherical:StereoMode>"
...
if opts.over_under:
  additional_xml = (spherical_xml_contents % "over-under")